tcms/templates/include/menu_items.html

```diff
@@ -22,7 +23,11 @@
             {% endif %}
         </a>
         <ul class="dropdown-menu">
-            {% include "include/menu_items.html" with menu_items=item.1 dropdown_class="-submenu" %}
+            {% if item.1|length == 0 %}
+                <li>{% trans "No items" %}</li>
+            {% else %}
+                {% include "include/menu_items.html" with menu_items=item.1 dropdown_class="-submenu" %}
+            {% endif %}
```

Comment:
A better fix would be not to show the last drop-down at all.

Will also need tests.